webpack/styles/widgets/_forms.scss

```diff
@@ -0,0 +1,99 @@
+.form-group {
+	@extend .panel-group;
+	margin-top: 20px;
+	text-align: left;
+	margin-bottom: 20px;
+	width: 180px;
+	}
+	
+.form {
+	@extend .panel; 
+	border-radius: 0px;
+	border: 1px solid $light-gray;
+}	
+	
+.form-container {
+	@extend .panel-body;
+	border-bottom: 1px solid $light-gray;
+	font-weight: bold;
+	border-radius: 0px;
+	font-size: 14px;
+	color: $cdc-blue;
+	padding: 6px 8px 6px 10px;
+}
+.form-title {
+	border-bottom: 1px solid $light-gray;
+	padding-bottom: 10px;
+}
+.form-list-group {
+	@extend .list-group;
+	display: table;
+	width: 100%;
+	font-size: 12px;
+	margin-bottom: 0px;
+	
+}
+.form-list-group-item {
+	@extend .list-group-item;
+	text-align: center;
+	display: table-cell;
+	width: 50%;
+	font-size: 12px;
+	border: 0px;
+}
+.form-item-icon {
+	color: $dark-gray;
+}
+.form-item-value {
+	font-size: 28px;
+	
+}
+.form-item-title {
+	font-size: 14px;
+	margin-top: 0px;
+	
+}
+.form-info {
+	border-top: 1px solid $light-gray;
+	padding-top: 6px;
+	padding-bottom: 6px;
+	font-weight:normal;
+	color: gray;
+}
+.question-set-details {
+	@extend .panel-body;
+	padding: 0px;
+	background-color: $light-gray;
+	.list-inline {
+		margin-left: 0px;
+		}
+	.question-set-items {
+		
+		margin-bottom: 0px;
+		}
+		a {
+		color: $cdc-blue;
+		}
+		span {
+		padding: 8px 6px 0px 4px;
+		}
+}
+.question-number {
+	background-color: $cdc-blue;
+	color: white;
+	height: 32px;
+	width: 140px;
+	padding: 4px 10px 0px 10px;
+	border-bottom: 1px solid $cdc-blue;
+	border-bottom-left-radius: 4px;
+	font-size: 12px;
+	i {
+		padding: 0px 6px 0px 0px;
+		float: left;
+	}
+	p {
+		font-size: 16px;
+		font-weight: bold;
+		padding-top: 2px;
+	}
+}
```

Comment:
One quick note while I was looking over this PR I noticed this file looks like it might be using tabs / have some formatting issues with the brackets, might want to just do some quick cleanup on this file.